swagger.yaml

```diff
@@ -827,3 +889,29 @@ components:
         - AVAILABLE
         - LOCKED
         - RESERVED
+
+    Prescription:
+      type: object
+      description: Prescription object
+      required:
+        - prescriptionDate
+        - prescriber
+        - description
+        - validUntil
+      properties:
```

Comment:
It does not have any kind of id?

Reply:
Nope

Reply:
And I suppose one cannot be deducted from other details?